inst/app/modules/plots.R

```diff
@@ -0,0 +1,327 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~
+# UI Side ####
+#~~~~~~~~~~~~~~~~~~~~~~~~
+
+inputsUI <- function(id){
+  ns <- NS(id)
+  tagList(
+    radioGroupButtons(ns("score"), "Choice of scoring", choices = c("All"= "all", "Coverage" = "cover", "Difference" = "diff", "Repeat" = "repeats", "Taper" = "taper", "Trend" = "trend"), individual = F, direction = "vertical", justified = F, width = "100%",
+                      checkIcon = list(
+                        yes = icon("square-check"),
+                        no = icon("square"))),
+    shinyjs::hidden(div(id = ns("args1"),
+                        sliderTextInput(ns("sig"), "Choice of Significance for Trend", choices = seq(from = 0.01, to = 0.2, by = 0.01), selected = 0.1, grid = TRUE))),
```

Comment:
the range for significant of trend should go up higher (basically 0 to 1). 

that would be too many steps / inputs with a slider ... can we use a numericInput instead?